src/Main.java

```diff
@@ -1,6 +1,7 @@
 import java.util.Scanner;
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Welcome Users in my programe.__.");
```

Comment:
Please change text to 
```suggestion
        System.out.println("Welcome");
```